### DISCUSSION TITLE
Windows binary release

### DISCUSSION THREAD

**Discussion by GlassGruber** [devmatteini/dra#51] [Ideas]
Hey this is very handy!
Is it possible to release also a binary for Windows? 
Thanks! 🎩

**Comment by devmatteini** [comment]
Hi, glad you like dra!

Binary for Windows and MacOS are planned as soon as I have time to work on it the next weeks!

**Reply by GlassGruber** [reply]
Great to hear thank you!

**Comment by devmatteini** [comment]
Hi @GlassGruber!

In the [latest release](https://github.com/devmatteini/dra/releases/tag/0.4.0) (0.4.0) you can download the windows binary!

Let me know if there are any problems!

**Reply by GlassGruber** [reply]
Hey that's great thank you!
A minor observation, I see I can't download files with a full url like  

    dra download https://github.com/devmatteini/dra/

I have to remove `https://github.com/` to make it work

![image](https://user-images.githubusercontent.com/1172428/197424865-34798831-010e-42b0-93bb-1a2864aac8d3.png)

However the app is working fine! 
The last thing I'm experiencing is a minor graphical annoyance with my terminal (cmder/conemu on Win7).  
It's not the first time I see glitches like these happening with rust apps, I guess is expected as the OS is getting unsupported more and more. 

You can see a sample in the previous screenshot and in the following, printed lines are not redrawn but stacked one after the other, is there any log/debug I can provide? 

![image](https://user-images.githubusercontent.com/1172428/197425179-24433270-aef6-42d5-80bb-8be4d391738a.png)




Anyway thank you for your time!

**Reply by devmatteini** [reply]
Hi! 

Yes, you cannot pass the full url but just the `owner/repo`.

For the spinner that is not redrawing, I will try to see if I can do something about it.

If you use Windows `cmd.exe` default terminal does it also happen?

**Reply by GlassGruber** [reply]
> If you use Windows `cmd.exe` default terminal does it also happen?

Yes same thing.

**Reply by devmatteini** [reply]
I've tested `cmder` on Windows 10 and works fine

**Reply by devmatteini** [reply]
Hi @GlassGruber,

I don't have a Windows 7 machine to do some tests and I think it's fine to only support Windows >= 10 (Windows 7 reached end of life in 2020).

I'm sorry your experience will not be as good as the others 😔

**Reply by GlassGruber** [reply]
No need to worry at all, I'm expecting many more tools to just break or stop working on W7, it's unfortunate but reasonable. The important thing for anything is to be functional first, not pretty. ;)

Thank you for your time with this very minor issue!
